docs/quick-start-guide.md

```diff
@@ -13,31 +13,20 @@ For a more detailed guide on how to use, compose, and work with `SparkApplicatio
 
 ## Installation
 
-Before installing the Spark Operator, run the following command to setup the environment for the operator:
+To install the Spark Operator, use the Helm [chart](https://github.com/helm/charts/tree/master/incubator/sparkoperator). 
 
 ```bash
-$ kubectl apply -f manifest/spark-operator-rbac.yaml
-$ kubectl apply -f manifest/spark-rbac.yaml
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install incubator/sparkoperator
 ```
 
-This will create a namespace `sparkoperator`, setup RBAC for the Spark Operator to run in the namespace. It will also
-setup RBAC for driver pods of your Spark applications to be able to manipulate executor pods. 
-
-The Spark Operator optionally uses a [Mutating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
-for Spark pod customization. To install the Spark Operator **without** the mutating admission webhook on a Kubernetes cluster, run the following command:
+Installing the chart will create a namespace `spark-operator`, set up RBAC for the Spark Operator to run in the namespace. It will also set up RBAC for driver pods of your Spark applications to be able to manipulate executor pods. In addition, the chart will create a Deployment named `sparkoperator` in namespace `spark-operator`. The chart by default enables a [Mutating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) for Spark pod customization. To install the Spark Operator **without** the mutating admission webhook on a Kubernetes cluster, install the chart with the flag `enableWebhook=false`:
```

Comment:
I think it worths mentioning what additional resources, e.g., the webhook service and secret storing the x509 certificate, get created as a result of enabling the webhook here.

Reply:
Sorry for the delay. I added a sentence explaining those extra stuff that gets created.